src/Configuration/Config.AzureKeyVault/src/Microsoft.Extensions.Configuration.AzureKeyVault.csproj

```diff
@@ -1,22 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <Description>Azure KeyVault configuration provider implementation for Microsoft.Extensions.Configuration.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageTags>$(PackageTags);azure;keyvault</PackageTags>
-    <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <InternalsVisibleTo Include="Microsoft.Extensions.Configuration.AzureKeyVault.Tests" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Reference Include="Microsoft.Azure.KeyVault" />
-    <Reference Include="Microsoft.Azure.Services.AppAuthentication" />
```

Comment:
Can Microsoft.Azure.KeyVault and Microsoft.Azure.Services.AppAuthentication dependencies be removed from the eng/ files?

Reply:
Probably, I can do that shortly.